docs/02-module-block-2/02-learning-goals.adoc

```diff
@@ -57,19 +55,17 @@ tbd.
   that admit only one output per input, and relational properties that
   admit multiple outputs per input.
 
-[[LG-2-3]]
-==== LG 2-3: Specification Languages
+[[LG-2-5]]
+==== LG 2-5: Know specification languages
```

Comment:
I would be more precise here: 
`Know of at least three different specification languages and their distinguishing properties`
Then you can add tool support as additional text and maybe also list a few specification languages?